index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Salman Ahmad Khan's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <a class="logo" href="#home">My Logo</a>
+        <i class="fa-sharp fa-solid fa-bars"></i>
+      </nav>
```

Comment:
- [ ] The former reviewer requested that you wrap the `nav` tag around the _**hamburger**_ icon but you have wrapped it around the _**logo**_ and the _**hamburger**_ icon. Could you wrap it around the _**hamburger**_ button only? The `nav` tag should contain navigational links to other parts of the website or other sites. 👍 In the end, you should have something like this:

```HTML
<header>
  <a class="logo" href="#home">My Logo</a>
  <nav>
    <i class="fa-sharp fa-solid fa-bars"></i>
  </nav>
</header>
```